dspace-api/src/test/data/dspaceFolder/config/spring/uclouvain/uclouvain-metadata-enhancers.xml

```diff
@@ -19,6 +19,7 @@
          
         @Author Michaël Pourbaix <michael.pourbaix@uclouvain.be>
     -->
+    <bean class="org.dspace.uclouvain.itemEnhancer.poller.UCLouvainItemEnhancerUpdatePoller"/>
```

Comment:
Missing general configuration in `dspace/config/spring/uclouvain-metadata-enhancers.xml`